STYLE_GUIDE.md

```diff
@@ -0,0 +1,164 @@
+
+- For descriptions that include time ranges, separate the numbers with an en dash. Avoid extra words such as between or from n to n.
+     - Correct: It can take 5–10 minutes before logs are available.
+     - Incorrect: It can take between 5 and 10 minutes before logs are available.
+
+- Use numerals for all measurement-based references, including time. Include a space between the number and the abbreviation for the unit of measure.
+     - Correct:
+        - 100 GB
+        - 1 TB
+        - 3 minutes
+        - 12 subnets (8 public and 4 private)
+     - Incorrect
+        - One hundred GB
+        - 1TB
+
+### Formatting and organization
+
+- When referring to fields in the API, use the following format: time_field
+
+- The following guidelines apply to all list types:
+     - Make lists parallel in content and structure. Don’t mix single words with phrases, don’t start some phrases with a noun and others with a verb, and don’t mix verb forms.
+     - Present the items in alphabetical order if the order of items is arbitrary.
+     - Capitalize the ﬁrst letter of the ﬁrst word of each list item.
+     - If the list is simple, you don’t need end punctuation for the list items.
+     - If the list has a mixture of phrases and sentences, punctuate each list item.
+     - Punctuate each list item with a period if a list item has more than one sentence.
+     - Punctuate list items consistently. If at least one item in a list requires a period, use a period for all items in that list.
+     - Titles are optional for most lists. If used, the title comes after an introductory sentence.
+     - Titles are highly recommended for procedures. Avoid titles for bulleted lists.
+     - Introductory sentences are required for lists.
+     - Introductory sentences should be complete sentences.
+     - Introductory sentences should end with a period if the list has a title.
+     - Introductory sentences should end with a colon if the list does not have a title.
+     - Don’t use semicolons, commas, or conjunctions (like and or or) at the end of list items.
+
+- Start all task-based headings with an infinitive. For example: “Create an index”. For conceptual sections, use a noun phrase. For example: “Migration to OpenSearch 2.0”
+
+- Stacked headings should never appear in our content. Stacked headings are any two consecutive headings without intervening text. Even if it is just an introductory sentence, there should always be text under any heading.
+
+### Procedures
+
+A procedure is a series of numbered steps that a user follows to complete a specific task. Users should be able to scan for and recognize procedures easily. Make procedures recognizable by using the following:
+
+- Predictable content parts
+- Parallel language constructions
+- Consistent formatting
+
+Following is an example of procedure phrasing and formatting from Amazon EC2.
+
+![Procedure example]({{site.url}}{{site.baseurl}}/images/Procedures.PNG)
```

Comment:
This link did not work for me?

Reply:
It could be because the link is to Procedures.PNG, but the file name is procedures.PNG with a lowercase p.

Reply:
For Git base files, we can use the old-fashioned Jekyll `direct/to/path` method for images, vs the `siteurl` variable. Should be working now.